src/views/PasswordResetProcess.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "../styles/signup.module.css";
+import Svg from "../components/SvgMap";
+
+const PasswordResetProcess = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.side}>
+        <div className={styles.text_wrap}>
+          <h1>Barefoot Nomad</h1>
+          <p>Making company travel and accommodation easy and convenient.</p>
+        </div>
+      </div>
+      <Svg />
+      <div className={styles.forms}>
+        <div className={styles.text_wrap}>
+          <p className={styles.fs_confirm} style={{ marginTop: "360px" }}>
+            Please go to your email address to complete the process.
+          </p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default PasswordResetProcess;
```

Comment:
you didn't have to create a new view just to display a message

Reply:
@blaise82, I'm afraid that the styling won't be the same.